_episodes/06-libraries.md

```diff
@@ -20,24 +20,42 @@ keypoints:
 
 *   A *library* is a collection of files (called *modules*) that contains
     functions for use by other programs.
-    *   May also contain data values (e.g., numerical constants) and other things.
-    *   Library's contents are supposed to be related, but there's no way to enforce that.
+    *   May also contain data values (e.g., numerical constants), classes and other data structures.
+*   [Modules][python-modules] can be organized in (sub-)directories, 
+    being called then (sub-)[packages][python-packages].
 *   The Python [standard library][stdlib] is an extensive suite of modules that comes
     with Python itself.
 *   Many additional libraries are available from [PyPI][pypi] (the Python Package Index).
 *   We will see later how to write new libraries.
 
-> ## Libraries and modules
+### How to know which library provides the functionality we need?
+
+Documentation. And google.
+
+*   It all starts with at before mentioned Python's [standard library][stdlib], 
+    have a glance there to see what is available by default with your Python distribution.
+*   [Browse PyPI][pypi-browse] packages if have an idea of the subject the funcitonality you need fits.
+*   When googling for a funcionality, be explicit on what you want, chances are
+    your question has already been answered in a community site (e.g, StackOverflow)
+    or a blog. For example, suppose you have a huge data file to read and you want
+    to have a glance on it at first, explicit search for "how to sample large data file in python"
+    and see how well the internet answers you, with many alternatives to explore.
+*   Follow newsletter, blogs that explore Python libraries, [Python Modules of the Week][pymotw]
+    is a good one, for instance.
+
+> ## Libraries, modules and packages
 >
-> A library is a collection of modules, but the terms are often used
-> interchangeably, especially since many libraries only consist of a single
-> module, so don't worry if you mix them.
+> A library is a collection of modules and packages, but the terms are often used
+> interchangeably. There is no restriction on the number of modules or packages
+> a library may contain, neither how it should be organized. By all means,
```

Comment:
contain, "nor are there restrictions on" how it should be...  

"By all means" is an odd way to start this sentence.